setup.py

```diff
@@ -31,16 +31,26 @@
     pip install git+https://github.com/in-toto/in-toto@develop
     ```
 """
+import os
+import re
+
 from setuptools import setup, find_packages
 
-version = "0.4.0"
+
+base_dir = os.path.dirname(os.path.abspath(__file__))
+
+def get_version(filename="in_toto/__init__.py"):
+  with open(os.path.join(base_dir, filename), encoding="utf-8") as initfile:
+    for line in initfile.readlines():
+      m = re.match("__version__ *= *['\"](.*)['\"]", line)
```

Comment:
Would you mind adding a brief comment mentioning that the format of the version field is not verified?

Reply:
done!